### PR TITLE
German translation (WIP)

### PR DESCRIPTION
**Pull request by ghost**
Don't merge, it's unfinished.

### CODE REVIEW

**Comment by ghost** [comment]
I already tried it, German letters like Ä/Ö/Ü don't cause problems.

**Comment by ghost** [comment]
But I don't know why, I can't enable it with `!locales set de`, only with changing the fallback locale to de.

File: src/cmd/misc/ship/ship.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ exports.ship = {
         }
         
         if (ctx.mentions[0] && !ctx.mentions[1] && new RegExp(`<@!?${ctx.mentions[0].id}>`).test(ctx.args[0])) {
-            ctx.mentions[0] = ctx.mentions[0];
             ctx.mentions[1] = null;
+            ctx.mentions[0] = ctx.mentions[1];
```
Comment (sr229) [inline src/cmd/misc/ship/ship.js]: Can you put this on another CL? You're changing way beyond the scope of your PR.

**Review by sr229 — CHANGES_REQUESTED** [review]

File: src/cmd/misc/ship/ship.js (hunk 1 above)
Comment (ghost) [reply src/cmd/misc/ship/ship.js]: Yes, I didn't wanted to do it, i wnated to do that in sthe master branch of my repo

Comment (ghost) [reply src/cmd/misc/ship/ship.js]: Fixed.

**Comment by ghost** [comment]
The userinfo command isn't translatable.
